.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-max-line-length = 100
+max-line-length = 120
```

Comment:
I did, but `tests/test_client.py:749` doesn't get adjusted by `black`, I suspect due to the fact that the comment on the end (`: type: ignore`) doesn't get pushed on to the next lines for reasons that are probably obvious.

So it seemed to me that the easiest solution was to give flake8 a little bit of ullage space compared with black's configuration.

Reply:
Really? If I start from master and run `black --max-line-length 100 tests/test_client.py` then it does something sensible. Did you perhaps start with a line length of 120 and re-ran black with a length of 100? That might cause it to first flatten the whole thing into one line, then be unable to split it correctly again.

Reply:
In retrospect that's probably what happened, because I did start with 120 I think. I had `black` configured like in `katgpucbf`, but it was fighting with `isort` which was configured with 100, it took embarassingly long to figure out that this was the problem. Once I then changed black to 100, this was the result. Let me try again.